routes/api.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const app = express.App();
+const ctrl = require('../controllers');
+
+// Current path = /api/v1
+app.get('/subscriber', ctrl.auth.find);
+app.post('/subscriber', ctrl.auth.create);
+app.delete('/logout', ctrl.auth.delete);
```

Comment:
change to: 'ctrl.subscriber.index' for first, 'ctrl.subscriber.destroy' for 3rd/last.

Reply:
Fixed!